index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MyPortfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap">
+    <script src="https://kit.fontawesome.com/366c518b05.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="#" class="firstTitle">Welcome</a>
+            <a href="#" class="hamburgerButton"><i class="fa-solid fa-bars"></i></a>
+        </nav>
```

Comment:
Nice navbar 💯 but make sure to adjust it to match the design better, I noticed that the design doesn't include the `Welcome` text you added, although it looks nice, it kinda changes the dynamic of the layout 🎨  Take a look below 👇🏽 

## Figma

![Screenshot 2023-04-26 at 09 17 23](https://user-images.githubusercontent.com/45739963/234587213-18ad2216-9110-4044-bb7e-f7c5bd8dc50c.png)

## Your design

![Screenshot 2023-04-26 at 09 19 08](https://user-images.githubusercontent.com/45739963/234587644-07867dc3-be3c-48c1-bff7-b9c1ce0602ba.png)

PS: Avoid pushing to a repo after submitting for review to avoid any confusion, I was about to add a comment about the hamburger color but you fixed it 😉 Well done 👍🏽